spec/lib/api/redfish-1.0/system-spec.js

```diff
@@ -1234,13 +1224,8 @@ describe('Redfish Systems Root', function () {
     });
 
     it('should return a valid iDRAC sel log service entry collection', function() {
-        wsman.isDellSystem.withArgs('1234abcd1234abcd1234abcd').resolves({
-            node: node, isDell: true, isRedfishCapable: false
-        });
-
-        wsman.getLog.withArgs(node, 'SEL').resolves(wsmanSelLog);
-
-        return helper.request().get('/redfish/v1/Systems/' + node.id +
+        wsman.getLog.withArgs(sinon.match.any, 'SEL').resolves(wsmanSelLog);
+        return helper.request().get('/redfish/v1/Systems/' + dellNode.id +
                                     '/LogServices/sel/Entries')
```

Comment:
sel - 'SEL' or 'sel' - does it matter?  since in the other script, we are switching to 'caps'.